trunk/includes/pardot-forms-widget-class.php

```diff
@@ -108,9 +108,9 @@ static function plugins_loaded() {
 			$url = preg_match( $regex, $_SERVER['HTTP_REFERER'] ) ? $_SERVER['HTTP_REFERER'] : site_url();
 
 			/**
-			 * Connvert the POST_ROOT_URL into a regex (escape the '.'s)
+			 * Convert the INLINE_FORM_URL into a regex (escape the '.'s)
 			 */
-			$post_root_url_regex = str_replace( '.', '\.', Pardot_Settings::POST_ROOT_URL );
+			$post_root_url_regex = str_replace('.', '\.', Pardot_Settings::INLINE_FORM_URL);
```

Comment:
I changed the name of this constant to avoid confusion in the future.  `INLINE_FORM_URL` is a constant only used for creating inline forms, which a feature that was only partially implemented, but was buggy, so we stuck with using iframes.  Frankly, the code paths for inline forms should be removed in the future.